frontend/src/utils/format.js

```diff
@@ -8,6 +8,11 @@ function formatDate(date) {
   return moment.utc(date).format('YYYY-MMM-DD')
 }
 
+function formatTwoMonthDate(date) {
```

Comment:
could we rename this function to something more generic? e.g.: formatDateWithMonthNumber

Reply:
oh nvm, I think you mean two month = MM?